docs/LICENSE.md

```diff
@@ -0,0 +1,7 @@
+Copyright (C) 2015 by Haris Amin
```

Comment:
@swizzlr I've seen some other projects have a Contributors.MD with he list of the contributors. But Yeah I'm cool with "Swiftdocker maintainers/contributors"